client/client.go

```diff
@@ -112,6 +113,39 @@ func (client *Client) doSync(method, path string, query url.Values, body io.Read
 	return nil
 }
 
+type asyncResult struct {
+	Resource string `json:"resource"`
+}
+
+func (client *Client) doAsync(method, path string, query url.Values, body io.Reader) (string, error) {
+	var rsp response
+
+	if err := client.do(method, path, query, body, &rsp); err != nil {
+		return "", fmt.Errorf("failed to communicate with server: %v", err)
```

Comment:
I was about to protest because obviously we'd used "unable to" nearly everywhere, but then I grepped:
```
john@fogey:~/canonical/snappy/src/github.com/ubuntu-core/snappy$ \egrep --include \*.go -r '"unable to' | wc -l
54
john@fogey:~/canonical/snappy/src/github.com/ubuntu-core/snappy$ \egrep --include \*.go -r '"failed to' | wc -l
51
john@fogey:~/canonical/snappy/src/github.com/ubuntu-core/snappy$ \egrep --include \*.go -r '"cannot' | wc -l
46
```

so, fair enough. `cannot` it is.